experiment/autobumper/bumper/bumper.go

```diff
@@ -286,6 +287,20 @@ func Run(o *Options) error {
 			}
 		}
 
+		// Check to see if the proper fork exists and if it does not, create one.
+		// TODO(mpherman): Handle case where account has repo of same name as one we are trying to autobump but is not a fork of it (e.g. test-infra)
+		_, err := gc.GetRepo(o.GitHubLogin, o.GitHubRepo)
+		if err != nil {
+			if err.Error() == getRepo404ErrMsg {
```

Comment:
Please write [left-aligned code](https://blog.codinghorror.com/flattening-arrow-code/) to avoid hard to read passages like this, i.E. here:
```
if err.Error() != getRepo404ErrMsg {
  return fmt.Errorf("failed to check")
}
// create fork, need need for an else clause
```

Reply:
Thanks for the link, that's pretty useful to learn